contribute-mozilla.html

```diff
@@ -48,7 +48,7 @@ <h2 style="color: #000;">Firefox</h2>
       </section>
       <section>
         <h2>¿Cómo empezar a contribuir a Mozilla?</h2>
-        <a class='fragment' href='https://contribute.mozilla.org'>contribute.mozilla.org</a>
```

Comment:
Invalid link.

Reply:
Thanks for catch this @jotisa.

The problem here is the protocol.
http://contribute.mozilla.org is redirecting to https://www.mozilla.org/contribute/, but https://contribute.mozilla.org doesn't resolve.

This site have a general information about contributing, and have some links to activate, donate, support, etc..

https://activate.mozilla.community/ is focused more in activities and events, this site will be migrating to the new community site (https://community.mozilla.org/) from march and start redirecting to there from april.

I think we should update the link to https://www.mozilla.org/contribute/
Aside to update the href attribute you have to update the text content to the new link too.